test/files/neg/t6446-additional.check

```diff
@@ -1,27 +1,28 @@
-    phase name  id  description
-    ----------  --  -----------
-        parser   1  parse source into ASTs, perform simple desugaring
-         namer   2  resolve names, attach symbols to named trees
-packageobjects   3  load package objects
-         typer   4  the meat and potatoes: type the trees
-superaccessors   5  add super accessors in traits and nested classes
-        patmat   6  translate match expressions
-    extmethods   7  add extension methods for inline classes
-       pickler   8  serialize symbol tables
-     refchecks   9  reference/override checking, translate nested objects
-       uncurry  10  uncurry, translate function values to anonymous classes
-        fields  11  synthesize accessors and fields, add bitmaps for lazy vals
-     tailcalls  12  replace tail calls by jumps
-    specialize  13  @specialized-driven class and method specialization
- explicitouter  14  this refs to outer pointers
-       erasure  15  erase types, add interfaces for traits
-   posterasure  16  clean up erased inline classes
-    lambdalift  17  move nested functions to top level
-  constructors  18  move field definitions into constructors
-       flatten  19  eliminate inner classes
-         mixin  20  mixin composition
-       cleanup  21  platform-specific cleanups, generate reflective calls
-    delambdafy  22  remove lambdas
-           jvm  23  generate JVM bytecode
-       ploogin  24  A sample phase that does so many things it's kind of hard...
-      terminal  25  the last phase during a compilation run
+     phase name  id  description
+     ----------  --  -----------
+         parser   1  parse source into ASTs, perform simple desugaring
+          namer   2  resolve names, attach symbols to named trees
+ packageobjects   3  load package objects
+          typer   4  the meat and potatoes: type the trees
+postTyperChecks   5  validate the result of typer
```

Comment:
A small thing, but it would be nice to not increase the max phase name length. (Output is shifted right here.) Also keep to convention of lowercase? "typerchecks".